tests/excel/test_excel_integration.py

```
@@ -0,0 +1,288 @@
+                block_text="",
+                state=None,
+                state_machine=None,
+            )
+            directive.env = env
+            directive.excel_loader = ExcelDataLoader(self.temp_dir)
+
+            xlsx_data = directive._load_json_data()
+            assert len(xlsx_data) == 2
+            assert xlsx_data[0]["Name"] == "Alice"
+
+        # .xls ファイルのテスト（パンダスがサポートしている場合）
```

Ruff (0.11.9)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)